cmd/fyne_demo/tutorials/canvas.go

```diff
@@ -45,5 +45,15 @@ func canvasScreen(_ fyne.Window) fyne.CanvasObject {
 		canvas.NewRasterWithPixels(rgbGradient),
 		gradient,
 		canvas.NewRadialGradient(color.NRGBA{0x80, 0, 0, 0xff}, color.NRGBA{0, 0x80, 0x80, 0xff}),
+		&canvas.Rectangle{
+			FillColor:   color.NRGBA{R: 255, G: 200, B: 0, A: 180},
+			StrokeColor: color.NRGBA{R: 255, G: 120, B: 0, A: 255},
+			StrokeWidth: 4.0,
+			Radius:      20},
+		&canvas.Rectangle{
+			FillColor:   color.NRGBA{R: 255, G: 200, B: 0, A: 180},
+			StrokeColor: color.NRGBA{R: 255, G: 120, B: 0, A: 255},
+			StrokeWidth: 2.0,
+			Radius:      45},
```

Comment:
Would it not make more sense to place these after the rectangle? Also, the second one just looks like another circle. Do we need to showcase both (the other ones are only one for each type of canvas object)?